### DISCUSSION TITLE
Bindings for Pytorch 1.13.0, 1.13.1 and 2.0.0

### DISCUSSION THREAD

**Discussion by carlosuc3m** [bytedeco/javacpp-presets#1352] [General]
Hello,
I am developing a library that uses JAvacpp to load different versions of Pytorch.
Until recently I was using your snapshots for pytroch 1.13.0 and 1.13.1, but they seem not to be available anymore.
Is it ok for you if I put those bindings in zenodo and download them from there? https://sandbox.zenodo.org/record/1200598#.ZFt1SKVBzIU

Also, is Pytorch 2.0 stable version going to be released? If not can I do the same for those bindings?
Regards,
Carlos

**Comment by saudet** [comment]
You can do whatever you want with the binaries, but you might as well put them on a Maven repo, no?
Why do you want to use Zenodo?

> Also, is Pytorch 2.0 stable version going to be released? If not can I do the same for those bindings?

Probably, yes.

**Comment by carlosuc3m** [comment]
I used Zenodo in order to reference you as the author. I can put them in maven too.

> Probably, yes.

Great! Thank you!
